ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs

```
@@ -125,8 +125,8 @@ public async Task<CreateETRAMeetingActionResponse> CreateETRAMeeting(ETRAIssue m
             {
                 JObject tmiJObject = await CreateTenancyManagementInteractionJObject(meetingInfo, incidentid, ticketnumber);
 
-                try
-                {
+                //try
+                //{
```

Is this intentional?

---

Actually, I think it was me that commented this out :-)

---

The cause of the exception was being swallowed - we should probably revert this commit

---

I think it was in your code I haven't done any changes in exception catching/logging

---

reverted